srsue/src/stack/upper/tft_packet_filter.cc

```diff
@@ -38,66 +38,147 @@ tft_packet_filter_t::tft_packet_filter_t(uint8_t
   log(log_)
 {
   int idx = 0;
+  uint length_in_bytes;
```

Comment:
please make the length explicit, i.e. use `uint32_t` and init to `0`. I know it's only used in the IPv6 case but the `switch` is so long it is easily forgotten